elfi/methods/utils.py

```diff
@@ -455,3 +455,46 @@ def flat_array_to_dict(names, arr):
     for ii, param_name in enumerate(names):
         param_dict[param_name] = np.expand_dims(arr[ii:ii + 1], 0)
     return param_dict
+
+
+def resolve_sigmas(parameter_names: List[float],
```

Comment:
are the parameter names not type `List[str]`?

Reply:
yes, of course